code/modules/mob/living/emote.dm

```diff
@@ -546,3 +546,9 @@
 	key_third_person = "exhales"
 	message = "breathes out."
 	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/living/swear
+	key = "swear"
+	key_third_person = "swears"
+	message = "says a swear word!"
```

Comment:
Honestly, this should make the player blurt out random swear words and insults (that do not go against the ToS obviously) ranging from mild cussing to sailor swearing and the [ultimate wizard swear](https://youtu.be/TqTHmzMk0Cw?t=228), otherwise what's the point of this PR when I could just use the say command?